index.html

```diff
@@ -0,0 +1,189 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <div class="Left_side">
+      <a href="#">
+        <div class="logo">
+            My logo
+        </div>
+    </a>
+    </div>
+    <div class="Right_side">
+       <nav>
+        <div class="bar">
+            <i class="fa fa-bars" aria-hidden="true"></i>
+        </div>
+       </nav>
+    </div>
+</header>
+<section class="container">
+  <div class="me">
+    <h1>I’m Mahram Hossain.</h1>
+    <span>Glad to see you!</span>
+    </div>
+        <div class="bio">
+         <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+         </div>
+        <div class="lets_contact">
+          <div class="contact">
+            <h2>LET’S CONNECT</h2>
+        </div>
+       
+            <section class="social_media">
+                <ul class="social-icons-set">
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+                  </li>
+                  <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                        aria-label="Medium"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+                  </li>
+                </ul>
+              </section>
+            </section>
+          </section>
+    </div>
+    <section class="middle_section">
+        <div class="pic">
+    
+        </div>
+        <div class="name">
+            <h1>
+                Mahram Hossain
+            </h1>
+        </div>
+        <div class="descript">
+            <div class="camp">
+                <h5>
+                    CANOPY
+                </h5>
+            </div>
+            <div class="year">
+                <ul class="list">
+                    <li>Back And Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+        </div>
+        <div class="task">
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
```

Comment:
- [ ] As stated by the previous reviewer, you have just one card in the work section compared to `4` in the project. Kindly add them up to stick to the user requirements 👍